Sasquatch/SasquatchObjC/AppCenterDelegateObjC.m

```diff
@@ -70,6 +70,10 @@ - (void)startAnalyticsFromLibrary {
   [MSAppCenter startFromLibraryWithServices:@[ [MSAnalytics class] ]];
 }
 
+- (void)setUserId:(NSString *)userId {
+  // TODO
```

Comment:
TODO with descriptions or you can write the actual code and comment it out. If someone else address this TODO, we need to know what to do here.